gapis/config/config.go

```diff
@@ -22,6 +22,7 @@ const (
 	DebugDeadCodeElimination   = false
 	LogExtrasInTransforms      = false // Logs all commands' extras together with transforms
 	LogMemoryInExtras          = false // Logs all commands' read/write memory observation together with extras
+	LogMappingsToFile          = false
```

Comment:
We should at least have a comment here that
1) This only works for Vulkan
2) Describes exactly what this transform is printing.
   (E.g. the remapping state at the end of the program)
Since 2 handles can be re-used it only shows you what the final remapping table looks like, and won't hold onto intermediates.